src/software/geom/rectangle.h

```diff
@@ -0,0 +1,242 @@
+#pragma once
```

Comment:
Not sure why this file is showing up in the diff. May need to merge again.

Reply:
removed